mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageDAO.java

```diff
@@ -197,7 +197,7 @@ private ByteBuffer toByteBuffer(InputStream stream) throws IOException {
             ids -> FluentFutureStream.of(
                 ids.stream()
                     .map(id -> retrieveRow(id, fetchType)
-                        .thenApply(resultSet ->
+                        .thenApply((ResultSet resultSet) ->
```

Comment:
https://github.com/linagora/james-project/pull/818/commits/c825214a49c37b6771e9c6aac2c1d0c96955af39 is the history just below quynh approuval.

![screenshot from 2017-06-01 14-53-40](https://cloud.githubusercontent.com/assets/6928740/26669990/1d04075a-46da-11e7-9f50-829381969dcb.png)